sections/dashboard/PortfolioChart/PortfolioChart.tsx

```diff
@@ -37,6 +37,7 @@ const Chart = styled.div`
 	min-width: 915px;
 	border: 1px solid #353333;
 	border-radius: 16px;
+	height: 200px ;
```

Comment:
Is there a reason to add this height back in before we have a portfolio chart ready?

Reply:
just gives it some balance until then